include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -462,6 +462,21 @@ class CommonOperatorLoader : public ProtobufLoader {
     return true;
   }
 
+  bool loadExpandDims(const OpType &op, const ArgumentDictionaryTy &dict) {
+    auto in = getNodeValueOrCreateConstantByName(op.input(0));
+    auto dims = dict.find("dims");
+
```

Comment:
Very small nit: would you mind removing the unnecessary blank lines here?  (Reasonable people can disagree on this, but I like to take maximal  advantage of my vertical screen real-estate!)